main.py

```diff
@@ -286,6 +319,44 @@ def validateSettings(staticLetters) -> bool:
     return True
 
 
+def normalized(sett: dict) -> list:
+    total = sum(sett[i] for i in sett)
+    return [(sett[i]/total, i) for i in sett]
+
+
+def orderLetters(configs: tuple = BIGRAMS_CONFIGS) -> str:
+
+    # Generate monograph sets for each language
+    monoSets = []
+    for config in configs:
+        if config.weight <= 0:
+            continue
+        with open(config.path, 'r') as biFile:
+            bis = {line[:2]: float(line[3:]) for line in biFile}
+        notNorm = {}
+        for bi in bis:
+            for char in bi:
+                if char in AUTO_LAYER_IGNORE:
+                    continue
+                if char in notNorm:
+                    notNorm[char] += bis[bi]*config.weight
+                else:
+                    notNorm[char] = bis[bi]*config.weight
+        monoSets.append(normalized(notNorm))
+
+    # Combine the monograph sets into one
+    combinedSet = {}
+    for monos in monoSets:
+        for freq, char in monos:
+            if char in combinedSet:
+                combinedSet[char] += freq
+            else:
+                combinedSet[char] = freq
+
+    # Sort the combined monograph set
+    return ''.join(sorted(combinedSet, key=lambda i: combinedSet[i], reverse=True)).lower()
```

Comment:
Please move the usage of `.lower()` a few lines up, using it to immediately lower bigrams when first reading them.
For an example, see [this line of code](https://github.com/Glitchy-Tozier/8vim_keyboard_layout_calculator/blob/main/main.py#L424).

Reply:
Done in 444e09cfcf29a2dc6c1ad61332d4f2e98f975ae6